packages/model/Table.ts

```diff
@@ -18,7 +19,7 @@ export interface Table extends RdfResource {
   csvMapping: Link<CsvMapping>
   identifierTemplate: string
   color: string
-  columnMappings: ColumnMapping[]
+  columnMappings: Array<Link<ColumnMapping>>
```

Comment:
In the UI, they are not really `Link`s right? I see that you fixed it by "convincing" typescript that they are full resources but I wonder if we are not already hitting some of the limits of sharing these models.

Reply:
This is just annotation so yes, the properties are actual resource objects.

That aside, I have been thinking about this and it is true that this is a contention point between the two usages of a shared model. In the long run I would rather that the UI model interface are for example generated from a SHACL shapes.

This would especially useful for the designer view.

Reply:
Yeah... let's not go into this for now ;)